uploads/assignment/run.py

```diff
@@ -225,13 +225,8 @@ def run(self):
             result = r.json()
             score = result['message']
 
-            if (result['status'] == 200):
```

Comment:
Due to the above, this condition might actually be needed.

Reply:
@recluze Yeah.

Reply:
Due to the above condition, we won't get here unless status['code'] is 200.

Response is returned when status_code is 200 else script exists. 

@recluze

Reply:
Yep, ok. Let's keep your method but in case of errors, show a highlighted output (kind of like I did with `=` signs. Just to make sure the student knows something went wrong. You can commit everything to master.